hologram_dashboard/fetcher/proto/enums.proto

```diff
@@ -0,0 +1,59 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+syntax = "proto2";
+
+package wireless_android_play_analytics;
+
+enum DataType {
```

Comment:
For proto definition, we need to specify the net tag number to use. For example,

// Next tag: 17
enum DataType {


}